JobConfig/cosmic/cosmic_s2_resampler_all.fcl

```diff
@@ -0,0 +1,8 @@
+// Re-sample cosmic dsstops particles from stage-1
```

Comment:
What is the value added by this script over its parent?  It looks like a test file.  I suspect the hard-coded input will go out of date quickly.

Reply:
I'll configure cosmic_s2_resampler.fcl as cosmic_s2_resampler_all.fcl and delete cosmic_s2_resampler_all.fcl
I prefer to keep a sample of input file for debugging purposes.
I can update the input file if it goes out of date.